cove/management/commands/base_command.py

```diff
@@ -19,28 +20,37 @@ def __init__(self, *args, **kwargs):
         super(CoveBaseCommand, self).__init__(*args, **kwargs)
 
     def add_arguments(self, parser):
-        parser.add_argument('file', help='File to be processed by Cove')
+        parser.add_argument('files', help='Files to be processed by Cove', nargs='+')
         parser.add_argument('--output-dir', '-o', default='', help='Directory where the output is created, defaults to the name of the file')
         parser.add_argument('--delete', '-d', action='store_true', help='Delete existing directory if it exits')
         parser.add_argument('--exclude-file', '-e', action='store_true', help='Do not include the file in the output directory')
+        parser.add_argument('--stream', '-st', action='store_true', help='Do not include the file in the output directory')
+
+    def handle(self, files, *args, **options):
+
+        stream = options.get('stream')
+        if (len(files) > 1 or len(glob.glob(files[0])) > 1 or os.path.isdir(glob.glob(files[0])[0])) and not stream:
```

Comment:
```if len(files) > 1 and not stream:``` should be enough.

Also, ```len(glob.glob(files[0]) > 1``` and ```os.path.isdir(glob.glob(files[0])[0])``` don't make much sense. @yolile I can see that perhaps you misunderstood what ```glob.glob``` does?